code/game/objects/structures/crates_lockers/largecrate.dm

```diff
@@ -64,7 +64,7 @@
 	M.animation_attack_on(src)
 	unpack()
 	M.visible_message(SPAN_DANGER("[M] smashes [src] apart!"), \
-					  SPAN_DANGER("You smash [src] apart!"), 5, CHAT_TYPE_XENO_COMBAT)
+					  SPAN_DANGER("You smash [src] apart!"), 5, message_flags = CHAT_TYPE_XENO_COMBAT)
```

Comment:
Uh wait, there's more, this uses the /mob variant of visible_message() which has a different signature

`/mob/visible_message(message, self_message, blind_message, max_distance, message_flags = CHAT_TYPE_OTHER)`

you need to add the blind_message or a null in there to match, it's all off by one, not juste the message_flags